### DISCUSSION TITLE
REQUEST : bazarr

### DISCUSSION THREAD

**Discussion by Patatra** [tteck/Proxmox#1300] [Ideas]
Hi,

Thanks for all those scripts. Could you add one for bazarr, a subtitle management.

Thanks!

**Comment by gitrhys** [comment]
I would also appreciate this script.

**Comment by ghost** [comment]
WOULD LOVE THIS ONE +1!!

**Comment by norbertjoni** [comment]
It would be nice, but in Alpine Linux, if it's possible.

https://github.com/morpheus65535/bazarr

also, would it be possible to make Alpine based for light apps for *arr-s?
